kernel/main.c

```diff
@@ -340,6 +353,52 @@ async_init(void)
       panic("Unable to create a kthread for do_async_init()");
 }
 
+static void
+setup_uefi_runtime_services() {
+   EFI_RUNTIME_SERVICES *RT = TO_PTR(uefi_rt_addr);
+
+   printk("Runtime Services: %p\n", RT);
+
+   struct mem_region ma;
+
+   EFI_MEMORY_DESCRIPTOR *virt_map = kmalloc((size_t)get_mem_regions_count());
+   int num_entries_virt_map = 0;
```

Comment:
Cosmetic: no need for such long names for short-lived local variables. This is not BIG_ENTERPRISE_CODE (tm) :-) 

Something like `num_entries` is more than enough.